document-generator-core/src/main/java/uk/gov/companieshouse/document/generator/core/document/render/RenderDocumentOpenHttpConnection.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.companieshouse.document.generator.core.document.render;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+
+public interface RenderDocumentOpenHttpConnection {
```

Comment:
This is a generic interface (not specific to document rendering). Can you name it something generic, e.g. HTTPConnectionHandler please?